packages/client/hmi-client/src/components/workflow/tera-workflow-node.vue

```diff
@@ -4,31 +4,44 @@
 			<h5>{{ node.operationType }}</h5>
 		</header>
 		<section class="inputs">
-			<li v-for="(input, index) in node.inputs" :key="index">
-				<div class="port"></div>
+			<li v-for="(input, index) in node.inputs" :key="index" ref="inputs">
+				<div
+					class="port"
+					@click.stop="selectPort(input)"
+					@mouseover="mouseoverPort(index, true)"
+					@focus="() => {}"
+				></div>
 				{{ input.label }}
 			</li>
 		</section>
 		<slot name="body" />
 		<section class="outputs">
-			<li v-for="(output, index) in node.outputs" :key="index">
+			<li v-for="(output, index) in node.outputs" :key="index" ref="outputs">
 				{{ output.label }}
-				<div class="port"></div>
+				<div
+					class="port"
+					@click.stop="selectPort(output)"
+					@mouseover="mouseoverPort(index, false)"
```

Comment:
This feels a bit weird, you wouldn't care about the position of the port until you click on it right? 

So why bother with the mouseover-event? Isn't it just firing a lot of emits we don't necessarily need? 
Moreover, even if we need it, why wouldn't we just do `mouseoverPort(output)` instead of passing in the index and true|false?